example/lib/screens/pin_request_screen.dart

```diff
@@ -251,11 +249,8 @@ class NumPad extends StatelessWidget {
     return Expanded(
       child: Container(
         margin: EdgeInsets.all(2),
-        child: FlatButton(
-          height: 50,
```

Comment:
Not part of this PR but these magic numbers hurt my eyes.

Reply:
Funny enough, this height is exactly the same as not having it in the replacement TextButton.